examples/subplots_axes_and_figures/gridspec_nested.py

```diff
@@ -24,22 +24,16 @@ def format_axes(fig):
 
 gs00 = gridspec.GridSpecFromSubplotSpec(3, 3, subplot_spec=gs0[0])
 
-ax1 = plt.Subplot(f, gs00[:-1, :])
-f.add_subplot(ax1)
-ax2 = plt.Subplot(f, gs00[-1, :-1])
-f.add_subplot(ax2)
-ax3 = plt.Subplot(f, gs00[-1, -1])
-f.add_subplot(ax3)
-
-
-gs01 = gridspec.GridSpecFromSubplotSpec(3, 3, subplot_spec=gs0[1])
-
-ax4 = plt.Subplot(f, gs01[:, :-1])
-f.add_subplot(ax4)
-ax5 = plt.Subplot(f, gs01[:-1, -1])
-f.add_subplot(ax5)
-ax6 = plt.Subplot(f, gs01[-1, -1])
-f.add_subplot(ax6)
+ax1 = f.add_subplot(gs00[:-1, :])
+ax2 = f.add_subplot(gs00[-1, :-1])
+ax3 = f.add_subplot(gs00[-1, -1])
+
+# the following syntax is the same as above, except perhaps more convenient:
+gs01 = gs0[1].subgridspec(3, 3)
+
```

Comment:
Can you remove this newline and the one after `gs00` above? Or add a comment above `gs00`?
When I first read the comment about "same as above", I thought it was about `add_subplot`, but it's really talking about the `gridspec.GridSpecFromSubplotSpec(...)`.

Reply:
Rephrased to `# the following syntax does the same as the GridSpecFromSubplotSpec call above:`